apps/dcos_dns/src/dcos_dns_handler.erl

```diff
@@ -361,6 +362,33 @@ max_payload_size(
 max_payload_size(_DNSMessage) ->
     ?MAX_PACKET_SIZE.
 
+-spec(loadbalance(dns:message()) -> dns:message()).
+loadbalance(#dns_message{answers=Answers}=DNSMessage) ->
+    % NOTE: The order of CNAME records does matter.
+    {CNameRRs, RRs} =
+        lists:splitwith(
+            fun (#dns_rr{type = Type}) ->
+                Type =:= ?DNS_TYPE_CNAME
+            end, Answers),
+    Strategy = dcos_dns_config:loadbalance(),
+    RRs0 = loadbalance(Strategy, DNSMessage, RRs),
```

Comment:
instead of `loadbalance` shall we call it `order`?

Reply:
I've renamed it to `randomize`. Please let me know what you think.